bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -1,166 +1,438 @@
 {# This Source Code Form is subject to the terms of the Mozilla Public
- # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
-
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 {% extends "firefox/base-resp.html" %}
-
 {% block page_title_prefix %}{% endblock %}
+{% from "macros.html" import fxfamilynav, send_to_device with context %}
 {% block page_title %}{{ _('{product} — {channel} Notes ({version})')|f(product=release.product, channel=channel_name, version=release.version) }}{% endblock %}
-
 {% block body_id %}notes{% endblock %}
-{% block body_class %}fx-notes sky{% endblock %}
-
+{% block body_class %}fx-notes blank{% endblock %}
 {% block page_css %}
-  {% stylesheet 'firefox_releasenotes' %}
+{% stylesheet 'firefox_releasenotes' %}
 {% endblock %}
-
 {% block optimizely %}
-  {% if waffle.switch('firefox-releasenotes-optimizely') %}
-     {% include 'includes/optimizely.html' %}
-  {% endif %}
+{% if waffle.switch('firefox-releasenotes-optimizely') %}
+{% include 'includes/optimizely.html' %}
+{% endif %}
 {% endblock %}
-
 {# channel_name is for display purposes where needed.  #}
 {% set channel_name = channel_name|default('') %}
-
 {% block content %}
-<main role="main">
+<main role="main" class="release-notes">
   <article id="main-content">
-
-  {% block notes_header %}
+    {% block notes_header %}
     <header class="notes-head">
-    {% block notes_heading_primary %}
-      <h1>{{ _('Firefox {channel} Notes')|f(channel=channel_name) }}</h1>
-    {% endblock %}
-    {% block notes_heading_secondary %}
-      <h2>{{ _('Version {version}, first offered to {channel} channel users on {date}')|f(channel=release.channel, date=release.release_date|l10n_format_date, version=release.version) }}</h2>
-    {% endblock %}
-
-    {% block platform_switch %}
-    {% if equivalent_release_url %}
-      <aside id="platform">
-        <h4>{{ _('View notes for:') }}</h4>
-        <ul>
-        {% if release.product == 'Firefox' %}
-          <li><span>{{ _('Desktop') }}</span></li>
-          <li><a href="{{ equivalent_release_url }}">{{ _('Android') }}</a></li>
+      <h1>
+        {{ _('Release Notes') }} 
+      </h1>
+      <span class="release_blurb_wrapper">
+        {{ _('Release Notes tell you what’s new in Firefox. As always, we welcome your ') }}
```

Comment:
This string shouldn't be split like this, as it makes translation much more difficult when the sentence is broken in the middle. The whole thing can go in one <code>{% trans %}</code> block.